starter-code/src/functions-and-arrays.js

```diff
@@ -39,6 +90,24 @@ const wordsUnique = [
   'simple',
   'bring'
 ];
+function uniquifyArray(array) {
+  let newUniqueArray = [];
+  if (array.length < 1 || array === undefined || array === null) {
+    newUniqueArray = [];
+  } else if (typeof array === "string") {
+    newUniqueArray.push(array);
+  } else {
+    for (let i = 0; i < array.length; i++) {
+      if (newUniqueArray.indexOf(array[i])>-1) {
```

Comment:
You can also use if(!newUniqueArray.includes(array[i])){newUniqueArray.push(array[i])}